shell/renderer/electron_render_frame_observer.cc

```diff
@@ -63,7 +64,8 @@ ElectronRenderFrameObserver::ElectronRenderFrameObserver(
   // In Chrome, app regions are only supported in the main frame.
   // However, we need to support draggable regions on other
   // local frames/windows, so extend support beyond the main frame.
-  render_frame_->GetWebView()->SetSupportsDraggableRegions(true);
+  static_cast<blink::WebViewImpl*>(render_frame_->GetWebView())
+      ->SetSupportsDraggableRegions(true);
```

Comment:
This is now done via a `browser->Renderer` ipc, to achieve the equivalent you can move this under https://github.com/electron/electron/blob/f784ea6f4f98a12208c9dc42d7e437280e285a06/shell/browser/api/electron_api_web_contents.cc#L1775 and call `content::WebContents::FromRenderFrameHost(render_frame_host)->SetSupportsDraggableRegions(true);`

Reply:
@deepak1556 I've pushed 58c17ef8ef78c03fafadf46122dc7939140db742 to mirror your  suggestion and have two questions:

1. Does this revision look correct to you, e.g. does it matter whether `SetSupportsDraggableRegions()` is called before or after the `"frame-created"` event is emitted?
2. I appreciate this change since it avoids the first draft's `content::WebViewImpl` downcast wart. :tada: I wish I'd seen this. To improve my understanding for next time, could you walk me how you got to this section of code?